src/Microsoft.AspNet.Routing/IRouter.cs

```diff
@@ -9,6 +9,6 @@ public interface IRouter
     {
         Task RouteAsync(RouteContext context);
 
-        string GetVirtualPath(VirtualPathContext context);
+        VirtualPathData GetVirtualPath(VirtualPathContext context);
```

Comment:
This interface misses the xml doc like in the other sources in this repo. I think we can add it later when we clean up this repo later. It will make them more consistent.